File: kotlin/PlanesAndroid/app/src/main/java/com/planes/common/PlaneRoundOptions.java
package com.planes.common;

public class PlaneRoundOptions {

    public PlaneRoundOptions() {
        reset();
    }

    private void reset() {
        m_ComputerSkillLevel = 2;
        m_ShowPlaneAfterKill = false;
    }

    public int m_ComputerSkillLevel = 2; // 0 - EASY, 1 - MEDIUM, 2- ADVANCED
    public boolean m_ShowPlaneAfterKill = false;
}

File: kotlin/PlanesAndroid/app/src/main/java/com/planes_multiplayer/single_player_engine/PlaneRoundOptions.kt
package com.planes_multiplayer.single_player_engine

class PlaneRoundOptions {
    private fun reset() {
        m_ComputerSkillLevel = 2
        m_ShowPlaneAfterKill = false
    }

    var m_ComputerSkillLevel = 2 // 0 - EASY, 1 - MEDIUM, 2- ADVANCED

    var m_ShowPlaneAfterKill = false

    init {
        reset()
    }
}